examples/vision/image_classification_from_scratch.py

```diff
@@ -2,7 +2,7 @@
 Title: Image classification from scratch
```

Comment:
Change title to:
"[KerasCV] Image classification from scratch"

Reply:
Done!